clusters/build-clusters/02_cluster/openshift/ci-operator/registry-pull-credentials_secret_template.yaml

```diff
@@ -8,7 +8,11 @@ objects:
       openshift.io/description: |
         This secret contains the pull secrets for api.ci cluster and the local registry.
         It is used to pull images
-        * by the image clients in ci-operator pod: the local images synced by dptp-controller are eventually consistent only
+        * by image client: The local images synced by dptp-controller are eventually consistent only. Before it is synced,
```

Comment:
To be precise, what is eventually consistent is the replacement directives in the config, not the syncing itself(it has a _bit_ of a delay (some seconds) but that should be neglible)